ert_gui/simulation/view/realization.py

```diff
@@ -38,26 +38,66 @@ def __init__(self, width, height, parent=None) -> None:
         super(RealizationDelegate, self).__init__(parent)
         self._size = QSize(width, height)
 
+    # TODO: fix this for more than one job
+    def _getJobColor(self, node: Node) -> QColor:
+        for _, stage in node.children.items():
+            # print("1: ", stage)
+            for _, step in stage.children.items():
+                # print("node2: ", step)
+                for _, job in step.children.items():
+                    # print("node3: ", job.data)
+                    status = job.data["status"]
+                    # print("job ", s)
+                    if status == "Success":
+                        status = "Finished"
+                    return QColor(*REAL_STATE_TO_COLOR[status])
+
```

Comment:
I'd really like to move this to the `data()` method of the snapshot model. The dumber these views are, the more are we conforming to the way of the QT.

Conversely, doing this in the model, allows you to later do

```python
painter.setBrush(self.model().data(index.row(), index, role=RealJobColorHint) 
```

Reply:
Agree, 100.001%